web/admin/src/SurveyResults.js

```diff
@@ -147,7 +147,7 @@ class SurveyResults extends Component {
         if (idExists) {
           adjustedTitleForExport = data.id
         }
-        newItem[adjustedTitleForExport] = answer
+        newItem[adjustedTitleForExport.trim()] = answer.replace(/"/g, "'")
```

Comment:
It looks like the proper way to escape a `"` is `""`

https://stackoverflow.com/questions/17808511/properly-escape-a-double-quote-in-csv

Does the CSV generator not do this?  Maybe we can fix that.